docs/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix Jira tracker created and updated timestamps (OSIDB-14)
 - Fix errata created and updated timestamps (OSIDB-453)
 - Restrict write operations on placeholder flaws (OSIDB-388)
+- Make UUID constant whenever possible (OSIDB-392)
```

Comment:
I would maybe word this differently, something like "avoid recreating flaws on CVE id changes whenever possible" or something like that, to an end user the UUID is an implementation detail, what really changes is that flaws were being recreated unnecessarily.

When I read the PR title I thought there was something wrong with the UUIDs in the database (collision or something)

Reply:
Good point. I will reword it with your suggestion